spec/support/fixtures/test_models.rb

```diff
@@ -4,6 +4,8 @@ def self.table_name_prefix
   end
 
   class ClientRequest < ActiveRecord::Base
+    belongs_to :approving_official, class_name: User
```

Comment:
not all client requests have an approving official (eg: 18F does not). do we still want to include the relation here?